packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_range-slider.scss

```diff
@@ -14,108 +15,16 @@
 
     .widget-range-slider {
         margin-bottom: 16px;
-        padding: 4px 12px;
+        padding: 5px 10px;
 
-        .rc-slider-handle,
-        .rc-slider-dot-active {
+        .rc-slider-handle {
             border-color: $brand-default;
         }
 
-        .rc-slider.rc-slider-disabled {
-            background-color: transparent;
-        }
-
         .rc-slider.rc-slider-with-marks {
             padding-bottom: 25px;
         }
 
-        &.has-error {
-            .rc-slider-track,
-            .rc-slider-rail {
-                background-color: $brand-danger;
-            }
-        }
-
-        .rc-slider {
-            position: relative;
-            box-sizing: border-box;
-            width: 100%;
-            height: 14px;
-            padding: 4px 0;
-            border-radius: 6px;
-            touch-action: none;
-            -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-        }
-
-        .rc-slider-rail {
-            position: absolute;
-            width: 100%;
-            height: 4px;
-            border-radius: 6px;
-            background-color: #e9e9e9;
-        }
-
-        .rc-slider-track {
-            position: absolute;
-            left: 0;
-            height: 4px;
-            border-radius: 6px;
-        }
-
-        .rc-slider-step {
-            position: absolute;
-            width: 100%;
-            height: 4px;
-            background: transparent;
-        }
-
-        .rc-slider-handle,
-        .rc-slider-dot-active {
-            position: absolute;
-            width: 14px;
-            height: 14px;
-            margin-top: -5px;
-            margin-left: -7px;
-            cursor: grab;
-            border: 2px solid $brand-default;
-            border-radius: 50%;
-            background-color: #ffffff;
-            touch-action: pan-x;
-
-            &:focus {
```

Comment:
Is it fine to remove everything due to potential backwards compatibility?

Reply:
This code migrated to atlas_core. It's just styling, nothing more.

Reply:
Even if they're migrated towards atlas_core, we still need to keep them around as far as I remember. But if you did something similar to Slider, then it's fine cuz I'm not entirely sure about this.

Reply:
Checked, no impact on old widgets